src/server/accessors/OAuthApps.ts

```diff
@@ -0,0 +1,26 @@
+import { IOAuthApp, IOAuthApps } from '../../definition/accessors/IOAuthApps';
+import { OAuthAppsBridge } from '../bridges/OAuthAppsBridge';
+
+export class OAuthApps implements IOAuthApps {
+    constructor(private readonly oauthAppsBridge: OAuthAppsBridge) {}
```

Comment:
If you check the other accessors, they receive the `appId` during instantiation and keep it around to use in bridge calls. This allows them to not require an `appId` as a parameter in each of its methods

Reply:
I made it like this because I was thinking about a future dependency injection that does not work when you have a mutable parameter on the constructor.